pkg/olm/convert/annotations.go

```diff
@@ -0,0 +1,23 @@
+// Package registry contains the manually vendored type definitions from
+// the operator-framework/operator-registry repository.
+package convert
+
+// AnnotationsFile holds annotation information about a bundle
+type AnnotationsFile struct {
+	// annotations is a list of annotations for a given bundle
+	Annotations Annotations `json:"annotations" yaml:"annotations"`
+}
+
+// Annotations is a list of annotations for a given bundle
+type Annotations struct {
+	// PackageName is the name of the overall package, ala `etcd`.
+	PackageName string `json:"operators.operatorframework.io.bundle.package.v1" yaml:"operators.operatorframework.io.bundle.package.v1"`
+
+	// Channels are a comma separated list of the declared channels for the bundle, ala `stable` or `alpha`.
+	Channels string `json:"operators.operatorframework.io.bundle.channels.v1" yaml:"operators.operatorframework.io.bundle.channels.v1"`
```

Comment:
Hmm.. these concepts are to be supported in other types of bundles too. The same question from the previous thought, what is the right place to expose this if it is generic.